core/templates/dev/head/base.html

```diff
@@ -51,7 +51,11 @@
     <meta property="og:image" content="{{DOMAIN_URL}}/images/logo/288x288_logo_mint.png">
 
     <link rel="apple-touch-icon" href="/images/logo/favicon.png">
-    <title itemprop="name" ng-bind="pageTitle">{% block subtitle %}{% endblock %} - {% block maintitle %}Oppia{% endblock %}</title>
+    <!-- The title is binded to the rootScope and it will be replaced for its
+    translation when the page is loading. This is the only way to translate the
+    page title because the head of the file is outside the scope of any other
+    controller. -->
```

Comment:
I updated the comment explaining about strings and translations ids. You are right, if the title is a string it would not be translated.

How can I test that? If I use one of the pages like "Notifications" where the title is not a translation id, when the page is eventually translated the test is going to break. Otherwise, I should alter the template inside the test, before the page is loaded. I was looking for examples in the code and I couldn't find anything like that.